templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+<div class="lune-page lune-page-product {{classes}}">
+  <div class="md-col-9 mx-auto {{classes}}">
+
+    {{#hero}}
```

Comment:
all of this data should be wrapped inside a product object and this component should only be rendered if that object is present